translations/ai_rules.tex/pl.po

```diff
@@ -195,12 +195,12 @@ msgid ""
 "If an AI Hero is instructed to draw a card, they will draw and resolve \\textbf{another card} from the AI Deck.\n"
 "\n"
 msgstr ""
-"Kiedy Bohater AI \\textbf{Aktywuje} Jednostkę, dobierz Kartę AI \\index{Karta AI} i rozpatrz ją zanim Jednostka wykona ruch lub zaatakuje.\n"
-"Jeżeli Talia AI wyczerpie się podczas Walki, nie dobieraj Kart.\n"
-"Efekt każdej Karty AI zależy od \\hyperlink{Difficulty}{Poziomu Trudności} gry.\n"
-"Karta Mocy \\includesvg[height=10px]{\\svgs/might.svg} zostaje przy Jednostce do momentu gdy nastąpi pierwszy atak/obrona.\n"
-"Talia Zaklęć AI jest używana za każdym razem, gdy dobierana jest Karta Magii \\includesvg[height=10px]{\\svgs/magic.svg} z Talii AI.\n"
-"Jeżeli Bohater AI otrzyma polecenie dobrania Karty, dobierze i rozpatrzy \\textbf{kolejną Kartę} z Talii AI.\n"
+"Kiedy Bohater AI \\textbf{Aktywuje} Jednostkę, dobierz Kartę AI \\index{Karta AI} i rozpatrz ją przed aktywacją Jednostki.\n"
```

Comment:
Ta zmiana na pewno taka miała być? Czy w momencie brania karty nie nastąpiła już aktywacja? Dlaczego nie robimy jak w wersji en?

Reply:
wiersz 198 - wczoraj z @qwrtln szukaliśmy jak skrócić tą kwestię w poprzednim brzemieniu. Ja zasugerowałem zapis: "...przed ruchem/atakiem Jednostki", ale wtedy nie wchodził tekst jak powinien na stronę i mieliśmy niezgodną ilość stron w pliku. Ta wersja zaproponowana dała nam "sukces", a w sumie nie zmienia ona sensu zdania z wersji angielskiej. Tak myślę...

Reply:
Ok, ale czy jak dobieramy kartę, to już nie nastąpiła aktywacja?

Reply:
to może tak: "... ją przed atakiem/ruchem Jednostki"